api/app/models/concerns/spree/webhooks/has_webhooks.rb

```diff
@@ -42,7 +42,7 @@ def resource_serializer
       end
 
       def updating_only_timestamps?
-        saved_changes.present? && (saved_changes.keys - %w[created_at updated_at]).empty?
+        saved_changes && (saved_changes.keys - %w[created_at updated_at]).empty?
```

Comment:
@aleksandarpetrushev @sebastian-palma I think we should also include `deleted_at` in this list

Reply:
@damianlegawiec There is an issue with this. We have to change the condition.
1. When you soft delete, you update only the deleted_at column, hence no webhooks are triggered
2. Need to check if event is update, not default event, because on destroy there are no saved changes as well

I will open a PR